lib/aerospike/batch_read.rb

```diff
@@ -68,7 +68,7 @@ def self.ops(key, ops, opt = {})
     # For internal use only.
     def ==(other) # :nodoc:
       other && other.instance_of?(self.class) &&
-        @bin_names.sort == other.bin_names.sort && @ops.sort == other.ops.sort &&
+        @bin_names&.sort == other.bin_names&.sort && @ops == other.ops &&
```

Comment:
- Since bins and ops are mutually exclusive, `bin_names` could be nil.
- There is no implementation of `==(other)` on `Aerospike::Operation` for `@ops.sort` to work and I noticed that `BatchWrite` does not use `sort` on `ops` comparison: https://github.com/aerospike/aerospike-client-ruby/blob/444508dff0ccff4dfc3c9fc6997d1a6d66acdead/lib/aerospike/batch_write.rb#L43-L48